pyro/distributions/torch_distribution.py

```diff
@@ -5,6 +5,7 @@
 import torch
 from torch.distributions import biject_to, constraints, transform_to
 
+import pyro.distributions
```

Comment:
Hmm, maybe it would be a bit less circular to create a smaller depencency cycle? Does this work?
```py
import pyro.distributions.torch
```